### ISSUE TITLE
Add quote_type option

### ISSUE THREAD

**Issue by dpinol** [editorconfig/editorconfig#410]
I suggest adding quote_type as standard option as specified at https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#ideas-for-domain-specific-properties

imho it's a very useful option at least relevant for JS, TS & python.
Users are asking for its support in IDEs (eg. https://youtrack.jetbrains.com/issue/WEB-21157#focus=streamItem-27-3617910.0-0) and IDEs are implementing non standard codes.

**Comment by cxw42** [comment]
@dpinol If I am understanding correctly, this would set whether editors used single or double quotes when generating code.  It would not affect users' editing of existing code.  Is that right, or am I missing something?  Thanks!

**Comment by dpinol** [comment]
yes, for instance when automatically insert the path for import/include commands for symbols such as functions or classes.

**Comment by cxw42** [comment]
Sounds reasonable to me.  Are there other editors that would use this option?  Differences in behaviour we need to consider?
